Gruntfile.js

```diff
@@ -360,7 +360,7 @@ module.exports = function(grunt) {
   /**
    * Create custom task aliases and combinations.
    */
-  grunt.registerTask('compile-cf', ['bower:cf', 'concat:cf-less', 'concat:topdocIcons']);
+  grunt.registerTask('compile-cf', ['bower:cf', 'concat:topdocIcons']);
```

Comment:
If we're leaving the subtask in there, let's update the comment at `concat:cf-less` to explain that it's no longer/not currently used.